common/src/web_api.rs

```diff
@@ -385,3 +387,20 @@ impl PreKeyResponseItem {
         }
     }
 }
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct SignalMessages {
+    pub destination: uuid::Uuid,
+    pub timestamp: u64,
+    pub messages: Vec<SignalMessage>,
+    pub online: bool,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct SignalMessage {
+    pub r#type: u32,
```

Comment:
Can we change this name to something without special characters

Reply:
no, this is defined by signal